packages/next/server/config-shared.ts

```diff
@@ -139,7 +139,7 @@ export type NextConfig = { [key: string]: any } & {
     esmExternals?: boolean | 'loose'
     staticPageGenerationTimeout?: number
     isrMemoryCacheSize?: number
-    nftTracing?: boolean
+    outputFileTracing?: boolean
```

Comment:
We have some users relying on this now, so we should add a warning mentioning this has been renamed which we can leave a comment to remove after it's stable